{{cookiecutter.project_slug}}/Makefile

```diff
@@ -72,7 +72,11 @@ docs-preview: docs-build
 {%- elif cookiecutter.documentation_engine == 'sphinx' -%}
 .PHONY:docs-build
 docs-build:
-	sphinx-apidoc -o docs/_build {{ cookiecutter.package_slug }}
+	{% if cookiecutter.project_layout == "src" -%}
```

Comment:
instead of adding this `if` inside the target, could you move it to the global context, and add it as a variable, please?

maybe something like

```
{% if cookiecutter.project_layout == "src" -%}
PACKAGE_PATH="src/{{cookiecutter.package_slug}}"
{% else %}
PACKAGE_PATH="{{cookiecutter.package_slug}}"
% endif -%}
```

and inside the target you will just use the variable:

```
sphinx-apidoc -o docs/_build ${PACKAGE_PATH}
```